components/automate-deployment/pkg/server/server.go

```diff
@@ -1761,6 +1761,45 @@ func (s *server) ManifestVersion(ctx context.Context, d *api.ManifestVersionRequ
 	}, nil
 }
 
+// SetManifest sets the current package manifest
+func (s *server) SetManifest(ctx context.Context, req *api.SetManifestRequest) (*api.SetManifestResponse, error) {
```

Comment:
I think this can get people into a bad state. Perhaps we should limit it to the case where we're doing a restore. I think the case where autoupgrades are off and the deployment-service in the manifest is not older than the current running one.

Reply:
:thinking: Yeah, I suppose you could break yourself in some scenarios, but this was intended as a "cleaner" way to get the deployment-service into the state we want before we start a restore. The `chef-automate` CLI doesn't expose and way to do this, so they'd have to gRPC curl it. I think the chances of a person doing that are super low, but then again, who knows?

Reply:
i was more thinking for ourselves. We might forget in the future when it is safe to do something like that and end up making things sad. Also, another solution might be to not have SetManifest at all and pass it into the restore call?

Reply:
I thought about adding it to the restore call but that just pollutes it even more. I suppose we could tho.

Reply:
Went ahead and changed it, we'll see how it goes